plugins/colorschemes/ayu.nix

```diff
@@ -40,10 +40,11 @@ helpers.neovim-plugin.mkNeovimPlugin config {
   };
 
   extraConfig = cfg: {
-    extraConfigLuaPre = ''
+    colorschemes.ayu.config.init = ''
       local ayu = require("ayu")
       ayu.setup(${helpers.toLuaObject cfg.settings})
       ayu.colorscheme()
     '';
+    extraConfigLuaPre = cfg.config.final;
```

Comment:
I'm confused, why is this needed here? Is this because assigining `final` to `configLocation` is currently gated behind `callSetup`? (I don't think it needs to be, though)